Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyIntSliderCtrl.hxx

```diff
@@ -14,13 +14,29 @@
 #if !defined(Q_MOC_RUN)
 #include <AzToolsFramework/UI/PropertyEditor/PropertyIntCtrlCommon.h>
 #include <AzQtComponents/Components/Widgets/SliderCombo.h>
+#include <AzToolsFramework/UI/PropertyEditor/TextBoxLikePropertyCtrl.hxx>
 #endif
 
 namespace AzToolsFramework
 {
+    namespace Internal
+    {
+        struct PropertyIntSliderCtrl_DeferredTextboxLikeEdit_traits
+            : public DeferredTextboxLikeEdit_default_traits
+        {
+            template<class C>
+            static bool isUserEditing(const C* obj)
+            {
+                return obj->m_sliderCombo->spinbox()->hasFocus();
+            }
+        };
+    }
```

Comment:
suggestion: Even though this is internal it looks identical to the definition above (other than the name) - it might be worth just making this available in an internal namespace in the header so both places can use the same one?